go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/julienschmidt/httprouter v1.3.0
 	github.com/sirupsen/logrus v1.7.0
 	github.com/smartystreets/goconvey v1.6.4
+	github.com/solarwinds/grpchan v1.1.1-0.20201009132539-a2c6e0d00272 // indirect
```

Comment:
why it's in v1?

Reply:
I'm a newbie to go modules, forgot to cd to v2/ ;)

Reply:
will fix that along with updating to solarwinds/grpchan from master